content/en/docs/contribute/style/style-guide.md

```diff
@@ -38,29 +38,25 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 {{< comment >}}[If you're localizing this page, you can omit the point about US English.]{{< /comment >}}
 
-## Documentation content standards
+## What Kubernetes documentation does and doesn't contain
 
-Kubernetes documentation does not host certain types of content:
+Kubernetes documentation only hosts content from projects within the
+ `kubernetes` or `kubernetes-sigs` GitHub organizations or from current CNCF
+  projects.
```

Comment:
nit: Could provide a link to the list of current CNCF projects.

Reply:
@kbhawkey  - yes that was on my list of things to add.

Reply:
@kbhawkey added links to cncf projects and sandbox projects as well as how to find out if a cncf project has been archived